tests/Cargo.toml

```diff
@@ -12,7 +12,7 @@ required-features = ["failpoints"]
 [[test]]
 name = "integrations"
 path = "integrations/mod.rs"
-required-features = ["tikv/testexport"]
+required-features = ["raftstore/testexport", "tikv/testexport"]
```

Comment:
I think this break "make test" as testexport features is not specified in anywhere.

Reply:
I wasn't able to repro. 

```
 git show --stat | more
commit abe56a3534762081bbfa1010917c8fc8d595f947

$ EXTRA_CARGO_ARGS=--no-run make test &> /dev/null 

$ echo $?
0
```

Reply:
What I mean is the integration test cases are not run at all.